### ISSUE TITLE
Document how to listen to Symfony events

### ISSUE THREAD

**Issue by matks** [PrestaShop/docs#955]

**Comment by zuk3975** [comment]
its all in symfony docs, I wouldn't duplicate it. Unless you have something specific in mind :thinking:  ?
https://symfony.com/doc/3.4/event_dispatcher.html

**Comment by matks** [comment]
> its all in symfony docs, I wouldn't duplicate it. Unless you have something specific in mind 🤔 ?
> https://symfony.com/doc/3.4/event_dispatcher.html

Yes, even though it's in indeed in Symfony docs, I am not sure every PrestaShop developer knows about it. So I thought that we should document an example of what you can do with Symfony events, especially when looking to customize part of the BO behavior.

It would be a new entry in https://devdocs.prestashop.com/1.7/modules/concepts/symfony/https://devdocs.prestashop.com/1.7/modules/concepts/symfony/
